packages/@uppy/dashboard/src/index.js

```diff
@@ -176,8 +176,8 @@ module.exports = class Dashboard extends Plugin {
 
     if (callerPluginType !== 'acquirer' &&
         callerPluginType !== 'progressindicator' &&
-        callerPluginType !== 'presenter') {
-      const msg = 'Dashboard: Modal can only be used by plugins of types: acquirer, progressindicator, presenter'
+        callerPluginType !== 'editor') {
+      const msg = 'Dashboard: The Dashboard can only be used by plugins of types: acquirer, progressindicator, editor'
```

Comment:
if we're tweaking the wording here anyway 😇 
```suggestion
      const msg = 'Dashboard: can only be targeted by plugins of types: acquirer, progressindicator, editor'
```